### PR TITLE
JPhyloRef v0.4.0

### PR DESCRIPTION
**Pull request by gaurav**
This PR creates a new release for the changes made in PRs #60, #71 and #72. Once this PR is approved, I'll publish this to Maven and create a v0.4.0 tag.

### CODE REVIEW

**Review by hlapp — APPROVED** [review]
So I think the description you give here is confusing and not quite correct. What you publish here to master is changes you previously made on a release tag. The PRs you mention were already merged to master. Right?

**Comment by gaurav** [comment]
Not quite. This PR is intended to create a new Maven release (v0.4.0) for JPhyloRef that includes changes from the three PRs I mentioned. I haven't created a new v0.4.0 tag yet -- I usually wait for your approval of the release PR first, then publish to Maven, then create the tag. I've rewritten the PR description to reflect that -- does that make more sense? Since you've approved this PR, I'll go ahead with creating the Maven release, creating the tag and merging this PR.

**Pull request merged**
